### PR TITLE
Test

### PR DESCRIPTION
**Pull request by MercurialG**

### CODE REVIEW

File: src/App.vue [hunk 1]
```diff
@@ -1,8 +1,11 @@
 <script setup lang="ts">
-import TheNavbar from "./components/TheNavbar.vue";
-import TheDrawer from "./components/TheDrawer.vue";</script>
+// import TheNavbar from "./components/TheNavbar.vue";
```
Comment (rwlanup) [inline src/App.vue]: Why are these commented? Components are used in templates.

File: src/components/TheDrawer.vue [hunk 2]
```diff
@@ -1,7 +1,12 @@
 <script setup lang="ts">
 import { Drawer, DrawerContent } from "@progress/kendo-vue-layout";
 import { useLocalStorage } from "@vueuse/core";
-import { computed } from "vue";
+import { useRouter } from "vue-router";
+
+import { computed, ref } from "vue";
+
+// const router = useRouter();
```
Comment (rwlanup) [inline src/components/TheDrawer.vue]: router is commented but used in onSelect function.

File: src/pages/boards/[id].vue [hunk 3]
```diff
@@ -1,7 +1,73 @@
 <script setup lang="ts">
-import AppImageDropzone from "../../components/AppImageDropzone.vue";
+import { ref, toRefs } from "vue";
+// import AppPageHeading from "../../components/AppPageHeading.vue";
```
Comment (rwlanup) [inline src/pages/boards/[id].vue]: Why is it commented?

File: src/pages/boards/[id].vue [hunk 4]
```diff
@@ -1,7 +1,73 @@
...
+import BoardDragAndDrop from "../../components/BoardDragAndDrop.vue";
+// import { useAlerts } from "@/stores/Alerts";
```
Comment (rwlanup) [inline src/pages/boards/[id].vue]: remove if unnecessary

File: src/pages/boards/[id].vue [hunk 5]
```diff
@@ -1,7 +1,73 @@
...
+      id: uuidv4(),
+    };
+    tasks.value.push(taskWithTheId);
+    resolve(taskWithTheId);
+  });
+}
+
+const updateBoard = (b) => {
+  board.value = b;
+  // alerts.success("Board updated!");
```
Comment (rwlanup) [inline src/pages/boards/[id].vue]: same here

File: src/pages/boards/[id].vue [hunk 6]
```diff
@@ -1,7 +1,73 @@
...
+};
+
+const deleteBoardIfConfirmed = () => {
+  console.log("delete board");
```
Comment (rwlanup) [inline src/pages/boards/[id].vue]:
```suggestion
  // @todo Delete board logic goes here
  console.log("delete board");
```

File: src/pages/boards/index.vue [hunk 7]
```diff
@@ -1,42 +1,75 @@
...
+      },
+    }));
   },
-  {
-    id: "3",
-    title: "My Third Board",
-    order: "[]",
+}));
+
+// function createBoard() {
```
Comment (rwlanup) [inline src/pages/boards/index.vue]: why is it commented but used in template?

**Review by rwlanup — CHANGES_REQUESTED** [review]
Left some comments. Generally issues with commented code and also I doubt if this runs successfully. Can you please look into it if it runs successfully?

File: src/App.vue (hunk 1 above)
Comment (MercurialG) [reply src/App.vue]: Was commented in streams code, uncommented

File: src/components/TheDrawer.vue (hunk 2 above)
Comment (MercurialG) [reply src/components/TheDrawer.vue]: Was commented in streams code, uncommented

**Review by rwlanup — APPROVED** [review]

**Pull request merged**